testsuite/utils/src/main/resources/META-INF/keycloak-server.json

```diff
@@ -105,7 +105,9 @@
             "remoteStoreEnabled": "${keycloak.connectionsInfinispan.remoteStoreEnabled:false}",
             "remoteStoreHost": "${keycloak.connectionsInfinispan.remoteStoreServer:localhost}",
             "remoteStorePort": "${keycloak.connectionsInfinispan.remoteStorePort:11222}",
-            "hotrodProtocolVersion": "${keycloak.connectionsInfinispan.hotrodProtocolVersion}"
+            "hotrodProtocolVersion": "${keycloak.connectionsInfinispan.hotrodProtocolVersion}",
+            "embedded": "${keycloak.connectionsInfinispan.embedded:true}"
+}
```

Comment:
Redundant bracket here. Also it seems like this needs to be updated in `testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json` to be reflected in the undertow-mode in the base testsuite.